### ISSUE TITLE
Nuproj - Cannot find the packed file

### ISSUE THREAD

**Issue by hsinyij** [nuproj/nuproj#293]
Hi,

I tried to use nuproj to automatically generate the Nuget package with build (quickbuild/cloudbuild). There are some files which are packed with the Nuget package. I checked the generated Nuget package and the files are well packed and well placed as we expected. However, when we install that package and want to get the files, we use the macro to resolve to the nuget package as follows: $(PkgMicrosoft_NaturalLanguage_Toolkit_1_0_0_6). When we manually generated the Nuget package, Visual Studio can find the path for the installed nuget package for us and we can get the files correctly. But with the nuget packages generated by nuproj, it will return an empty string and that will result in a wrong path to the installed nuget package. In that case, we cannot get the files packed by that nuget package.

Please help me resolve this. Thanks a lot.

**Comment by AArnott** [comment]
@hsinyij Can you share the nuget package, or use the `tree /f` command on the unzipped directory from the package so we can see its layout?
Alternatively if you just tell us whether all the assemblies in the package are found under the lib folder, that can help.
My suspicion is that nuproj is building a typical NuGet package, but your $(Pkg...) macro makes me think you may be using the package as a glorified zip file and expecting the files to be laid out in some other way.

**Comment by hsinyij** [comment]
Hi,

Yes. I got all the .dlls under lib folder, but some data files which are needed is outside the lib folder. The structure is like:

DataFiles\
lib\
NativeDlls\

After I posted this question, I did search a little bit about that and found that there is a flag for nuproj: NoDefaultExcludes. I set it to true, rebuilt the nuget package and re-install that in my project. Then everything works for me. I am not sure which step resolved the issue though. But Thanks for the help anyways.
